### PR TITLE
bb terminal variants

### PR DESCRIPTION
**Pull request by robsideshow**
https://buganizer.corp.google.com/issues/174651983

Document the new terminal variants for the linux_terminal and windows_vm resources.  The actual diff is much smaller than it looks, since I shifted the position of the Windows VM section so all the BB resources are grouped together.

### CODE REVIEW

**Comment by robsideshow** [comment]
cc @supersam654 @jgan

File: lab-bundle-spec.md [hunk 1]
```diff
@@ -288,6 +288,11 @@ reference               | displayed as
 [DOMAIN].admin_password | copyable text
 
 
+
+
+
+#### BumperBoats Resources
```
Comment (bpg130) [inline lab-bundle-spec.md:294]: BumperBoats is an internal name and not something that I think we should publicize (similar to how Navy does not show up anywhere on this page).

I would either remove this header or rename it to something along the lines of `Virtual Machine Resources`.

File: lab-bundle-spec.md [hunk 2]
```diff
@@ -317,13 +322,46 @@ startup_script.path |          | path  | Relative path to a directory tree with
     startup_script:
       path: startup.sh
 ```
+###### Variants for Linux Terminal
+
+The allowed `linux_terminal` variants are:
+- it_cert [default]  (access to internet via shared Cloud NAT IP address)
```
Comment (bpg130) [inline lab-bundle-spec.md:328]: Nit: Maybe clarify inbound vs outbound internet access?

`outbound access to internet via shared Cloud NAT IP address, no inbound internet access`.

**Review by bpg130 — CHANGES_REQUESTED** [review]

File: lab-bundle-spec.md (hunk 1 above)
Comment (robsideshow) [reply lab-bundle-spec.md]: Makes sense, though it sort of renews my vague ongoing confusion about the intended audience/purpose of this document.

File: lab-bundle-spec.md (hunk 2 above)
Comment (robsideshow) [reply lab-bundle-spec.md]: how's that?

**Review by bpg130 — APPROVED** [review]

File: lab-bundle-spec.md (hunk 2 above)
Comment (bpg130) [reply lab-bundle-spec.md]: LGTM

**Pull request merged**